Form-Controls/index.html

```diff
@@ -17,11 +17,51 @@ <h1>Product Pick</h1>
         <!-- 
           try writing out the requirements first as comments
         this will also help you fill in your PR message later-->
+        <form>
+          <p>
```

Comment:
I don't think `p` tag is appropriate here. This tag is used for paragraphs of text and you have form field here. Please, update.